modules/encoders/cmd/base64.rb

```diff
@@ -48,7 +48,7 @@ def encode_block(state, buf)
     when 'base64'
       raise EncodingError if (state.badchars.bytes & '(|)'.bytes).any?
 
-      base64_decoder = '(base64 --decode || base64 -d)'
+      base64_decoder = '(base64 --decode||base64 -d)'
```

Comment:
I'm thinking someone might inadvertently add whitespace here again in the future unless there's a comment or automated test to ensure the whitespace is removed

Reply:
Added comment. See https://github.com/rapid7/metasploit-framework/pull/19993/commits/9a60caf36d18080180e1969a3119fb39064f5544.